students/prgrover/lesson03/mailroom.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+# Global tuple containing donor list and donation amounts
+donor_list = [["Harry Potter", [10000, 5000, 500.55]], ["Ronald Weasley", [2499.99, 7500.01]], ["Hermione Granger", [100, 2000, 30000]], ["Draco Malfoy", [10, 888.88]], ["Neville Longbottom", [10]]]
+
+# Email template for Thank You message
+email = """Dear {},\n\nThank you for your generous donation of ${}.\n\nSincerely,\nHogwarts School of Witchcraft and Wizardry"""
+
+
+def send_thankyou():
+    """
+    * If the user (you) selects ‘Send a Thank You’, prompt for a Full Name.
+    * If the user types ‘list’, show them a list of the donor names and re-prompt
+    * If the user types a name not in the list, add that name to the data structure and use it.
+    * If the user types a name in the list, use it.
+    * Once a name has been selected, prompt for a donation amount.
+    * Turn the amount into a number – it is OK at this point for the program to crash if someone types a bogus amount.
+    * Once an amount has been given, add that amount to the donation history of the selected user.
+    * Finally, use string formatting to compose an email thanking the donor for their generous donation. Print the email to the terminal and return to the original prompt.
+
+    Args: None
+    """
```

Comment:
Check out PEP 257 for Python multi-line doc string conventions: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings